### DISCUSSION TITLE
Import layout - custom series deleted upon changing data

### DISCUSSION THREAD

**Discussion by Kuto587** [facontidavide/PlotJuggler#855] [Q&A]
Hi,
I used plot juggler do visualize a couple of ulog files, which worked great and has some very awesome features.

I also configured a view to my liking (including several custom series, such as roll, pitch and some other transformation) and saved this layout. When I now load a _different_ ulog file (containing the same datastreams), the custom series dissapear from the list and also from the plots. Why? The datasource changed of course, but the datastreams stayed the same! If I change the line in the layout .xml file which specifies which data file is loaded, to some other ulog file, the custom transformations stay the same. But, of course, I do not want to copy the layout files each time I want to look at another plot. Am I overlooking something, or is this a bug?

**Comment by grandpajustin** [comment]
I have a similar question.  I would like to create a single format / layout / template into which I can load ulogs and have my custom timeseries, functions, and graphs populate.  These ulogs are created by a flight controller and have the same time series names.  Is there a way to do this?
